README.md

```diff
@@ -46,23 +49,35 @@ Launching a runtime and connecting
 ```javascript
 const { connect, Identity } = require("hadouken-js-adapter");
 
-connect({
-    uuid: "my-uuid-123",
-    runtime: {
-        version: 'stable'
-    }
-}).then(logic).catch(connError);
-
-function logic(fin) {
-    // Logic goes here
+async function launchApp() {
+    const fin  = await connect({
+        uuid: "my-uuid-123",
+        runtime: {
+            version: 'stable'
+        }
+    });
+
+    const version = await fin.System.getVersion();
+    console.log("Connected to Hadouken version", version);
+
+    const app = await fin.Application.create({
+        name: "adapter-test-app",
+        url: 'http://hadouken.io/',
+        uuid: "adapter-test-app",
+        autoShow: true,
+        nonPersistent : true
+    });
+
+    await app.run();
 }
 
-function connError(err) {
-
+launchApp().then(() => {
```

Comment:
Should we try to keep syntax consistent between async/await and promises or is this because it's top level?